thunder/tests/opinfos.py

```diff
@@ -1730,6 +1730,35 @@ def gen(op, device, dtype, requires_grad):
 elementwise_unary_ops.append(relu6_opinfo)
 
 
+def hardshrink_singularity_at(lambd):
+    return lambda a: torch.where(a >= 0, a - lambd, a + lambd)
+
+
+hardshrink_opinfo = OpInfo(
```

Comment:
Why separate the OpInfos? A comment would be helpful for that unusual pattern, too

Reply:
Yep!